region/lowernorfair/east/Lower Norfair Fireflea Room.json

```diff
@@ -612,6 +700,52 @@
         }}
       ]
     },
+    {
+      "link": [3, 2],
+      "name": "LN Firefleas Top to Bottom Itemless Damageless",
+      "notable": true,
+      "requires": [
+        {"or": [
+          "canInsaneWalljump",
+          "canInsaneJump"
```

Comment:
This strat was simply moved to the end of the [3, 2]'s. Then I converted the `canTrickyJump` to `canInsaneJump`. I figured this was accurate since it was an `Extreme` notable?